### PR TITLE
[Rule Tuning] Small update on rule descriptions

### PR DESCRIPTION
**Pull request by w0rk3r**
Incorporates the improvements from docs/package review.
Adds a threat mapping to an AWS Rule

### CODE REVIEW

File: rules/ml/ml_auth_rare_user_logon.toml [hunk 1]
```diff
@@ -9,10 +9,10 @@ anomaly_threshold = 75
 author = ["Elastic"]
 description = """
 A machine learning job found an unusual user name in the authentication logs. An unusual user name is one way of
-detecting credentialed access by means of a new or dormant user account. A user account that is normally inactive
-(because the user has left the organization) that becomes active may be due to credentialed access using a compromised
-account password. Threat actors will sometimes also create new users as a means of persisting in a compromised web
-application.
+detecting credentialed access by means of a new or dormant user account. An inactive user account that is normally
```
Comment (brokensound77) [inline rules/ml/ml_auth_rare_user_logon.toml:12]: inactive inactive 🙂 - should drop one

File: rules/windows/privilege_escalation_unusual_parentchild_relationship.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ language = "eql"
 license = "Elastic License v2"
 name = "Unusual Parent-Child Relationship"
 references = [
-    "https://github.com/sbousseaden/Slides/blob/master/Hunting MindMaps/PNG/Windows Processes%20TH.map.png",
+    "https://github.com/sbousseaden/Slides/blob/master/Hunting%20MindMaps/PNG/Windows%20Processes%20TH.map.png",
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_unusual_parentchild_relationship.toml:18]: I think there _may_ be more instances of this

Comment (brokensound77) [reply rules/windows/privilege_escalation_unusual_parentchild_relationship.toml:18]: just checked, this should be it

**Review by brokensound77 — APPROVED** [review]
One suggestion then LGTM

File: rules/windows/privilege_escalation_unusual_parentchild_relationship.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/privilege_escalation_unusual_parentchild_relationship.toml:18]: Also checked, could not find any other occurrence

**Pull request merged**